site/_posts/2017-10-18-vote-on-naming-h-release.html.md

```diff
@@ -0,0 +1,27 @@
+---
+title: Voting on Name of H Release
+author: chessbyte
+date: 2017-11-06 09:02:24 UTC
+tags: releases announcements
```

Comment:
the voting posts have so far been tagged with releases and collaboration. there's nothing to announce yet :)